src/toolbar/example/index.ts

```diff
@@ -0,0 +1,51 @@
+import { ProjectorMixin } from '@dojo/widget-core/mixins/Projector';
+import { w, v } from '@dojo/widget-core/d';
+import { WidgetBase } from '@dojo/widget-core/WidgetBase';
+import { WidgetProperties, TypedTargetEvent } from '@dojo/widget-core/interfaces';
+
+import Toolbar from '../Toolbar';
+import dojoTheme from '../../themes/dojo/theme';
+
+export class App extends WidgetBase<WidgetProperties> {
+	private _theme: {};
+
+	themeChange(event: TypedTargetEvent<HTMLInputElement>) {
+		this._theme = event.target.checked ? dojoTheme : {};
+		this.invalidate();
+	}
+
+	onAttach() {
+		const style = document.createElement('style');
```

Comment:
This method is purely for demo styling purposes. See above comment re: Toolbar and its children.